charts/timed/values.yaml

```
@@ -307,13 +305,19 @@ redmine:
   url: redmine.example.com
   # redmine.apiKey -- Redmine API Key
   apiKey: ""
-  # redmine.htaccessUser -- Redmine htaccess user
-  htaccessUser: ""
-  # redmine.htaccessPassword -- Redmine htaccess password
-  htaccessPassword: ""
   # redmine.spenthoursField -- Spent hours field id on Redmine
   spenthoursField: 0
 
+sentry:
+  # sentry.enabled -- Enable Sentry integration
+  enabled: false
+  # sentry.dsn -- Sentry DSN for error reporting. Set to enable Sentry integration
+  dsn: sentry.example.com
+  # sentry.tracesSampleRate -- Sentry trace sample rate, Set 1.0 to capture 100% of transactions
+  tracesSampleRate: 1.0
+  # sentry.sendDefaultPii -- Associate users to errors in Sentry
+  sendDefaultPii: true
```

Doesn't this need to be Python `True` (and quoted)?
```suggestion
  sendDefaultPii: "True"
```

---

Ahh this gets passed 1:1 as a env var, sure then.